core/src/test/scala/org/apache/spark/sql/delta/DeltaOptionSuite.scala

```diff
@@ -177,4 +179,62 @@ class DeltaOptionSuite extends QueryTest
       }
     }
   }
+
+  test("support no compression write option (defaults to snappy)") {
+    withTempDir { tempDir =>
+      val path = tempDir.getCanonicalPath
+      withTable("compression") {
+        spark.range(100)
+          .writeTo("compression")
+          .using("delta")
+          .option("compression", "snappy")
```

Comment:
this test says "support no compression write option" but here you are explicitly setting one using `.option`

Reply:
oops, fixed!